DocumentedExamples/Meridional_heat_transport.ipynb

```diff
@@ -5,15 +5,19 @@
    "metadata": {},
```

Comment:
<p>As I mentioned above, there are other differences here. Specifically, the <code>shflux</code> method assumes steady state in the OHC. Also, your current 3rd method as calculated above is different because it extracts only the non-reference-temperature dependent component of MHT (which differs from the other calculations both through the potential tendency of sea level at different locations and due to the presence of net surface volume flux at different latitudes).</p><p> </p><p>With regards to the spatially uniform specific heat capacity - this is not an approximation. The model uses Conservative Temperature as its prognostic temperature variable which has a constant heat capacity associated with it (effectively, Conservative Temperature is defined such that you can use a constant specific heat capacity, i.e., it is "conservative"). If you're interested in more details on this please see Trevor's recent paper in GMD (https://gmd.copernicus.org/articles/14/6445/2021/)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/285/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
Thanks for the reference, Ryan. I didn't know about it!